webapp/geek_text/templates/geek_text/base.html

```diff
@@ -19,18 +19,39 @@
 
 		<nav class="navbar navbar-inverse" style="background-image: -webkit-linear-gradient(top,#2d943a 0,#158223 100%);
 		    border-color: #2d943a;">
-		  <div class="container-fluid">
-		    <div class="navbar-header">
-		      <a class="navbar-brand" href="" style="color: #fff;">Geek Text</a>
-		    </div>
-		    <ul class="nav navbar-nav" >
-		      <li class="active" ><a href="/browse" style="background-image: linear-gradient(to bottom,#26632e 0,#26632e 100%);">Browse</a></li>
-		    </ul>
-		    <ul class="nav navbar-nav navbar-right">
-		      <li><a href="#"><span class="glyphicon glyphicon-user"></span>Account</a></li>
-		      <li><a href="#"><span class="glyphicon glyphicon-log-in"></span>Cart</a></li>
-		    </ul>
-		  </div>
+            <div class="container-fluid">
+                <div class="navbar-header">
+                    <a class="navbar-brand" href="" style="color: #fff;">Geek Text</a>
+                </div>
+                <ul class="nav navbar-nav" >
+                    <li class="active" ><a href="/browse" style="background-image: linear-gradient(to bottom,#26632e 0,#26632e 100%);">Browse</a></li>
+                </ul>
+                <ul class="nav navbar-nav navbar-right">
+
+                    {% if user.is_authenticated %}
+                        <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink"
```

Comment:
Why is there a dropdown now for all these pages? We only have a profile page for authenticated user anyway.
Please just keep it simple.